iosApp/Survey/Sources/Presentation/Modules/SurveySelection/SurveyItemView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SurveyItemView.swift
+//  Survey
+//
+//  Created by Bliss on 28/9/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Kingfisher
+import Shared
+import SwiftUI
+
+struct SurveyItemView: View {
+
+    let survey: SurveyUI
+
+    var body: some View {
+        ZStack {
+            GeometryReader { geo in
+                KFImage(URL(string: survey.largeImageUrl) ?? URL(fileURLWithPath: ""))
```

Comment:
May we we can turn this part - `URL(string: survey.largeImageUrl) ?? URL(fileURLWithPath: "")` into an extension as there are chances we will need to reuse for intializing the `URL` 🤔

Reply:
Updated in https://github.com/blyscuit/kmm-swiftUI-IC/pull/57/commits/de04d90f5ac9e5bbc3b76ee7f3127b8ae887468e